props/saber_BC_buttons.h

```diff
@@ -571,6 +611,10 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
         last_twist_ = millis();
         saber_off_time_ = millis();
         battle_mode_ = false;
+        if (alt_blade_on_) {
+          hybrid_font.DoEffect(EFFECT_USER4, 0);
+          alt_blade_on_ = false;
```

Comment:
I think this is confusing, since SB_Effect() will also modify alt_blade_on_.
I think you need to decide if SB_Effect is doing this, or if it should be done where DoEffect is called.

Reply:
I was under the impression that DoEffect triggers the blade animation, while SB_Effect plays the sound, no?